e2e/fixtures/fdb_cluster.go

```diff
@@ -405,7 +422,7 @@ func (fdbCluster *FdbCluster) UpdateClusterSpecWithSpec(desiredSpec *fdbv1beta2.
 		}
 		// Retry here and let the method fetch the latest version of the cluster again until the spec is updated.
 		return false
-	}).WithTimeout(1 * time.Minute).WithPolling(1 * time.Second).Should(gomega.BeTrue())
+	}).WithTimeout(5 * time.Minute).WithPolling(1 * time.Second).Should(gomega.BeTrue())
```

Comment:
Any reason to increate the timeout here?

Reply:
my test (when I taint 5 pods concurrently) fails at the timeout. When too many nodes tainted in a test, the cluster will keep changing its spec due to continuous replacements. 
So I increased the timeout and the test passed. 
Because this function is only used in test, I *think* it's safe. 

Open to other ideas.